packages/pyolite-kernel/py/pyolite/pyolite/__init__.py

```diff
@@ -20,6 +21,9 @@
 from .display import LiteStream
 from .interpreter import LitePythonShellApp
 
+if not os.environ.get("MPLBACKEND"):
+    os.environ["MPLBACKEND"] = "module://matplotlib_inline.backend_inline"
```

Comment:
Actually this may be conflicting with the following patch:

https://github.com/jupyterlite/jupyterlite/blob/f11fb9a05d56d69fa5ec6e9b04ab96a097f2a83a/packages/pyolite-kernel/py/pyolite/pyolite/patches.py#L5-L7